schemas/com.snowplowanalytics.snowplow/weather_enrichment_config/jsonschema/1-0-0

```diff
@@ -0,0 +1,52 @@
+{
+	"$schema": "http://json-schema.org/schema#",
+	"description": "Schema for weather enrichment",
+	"self": {
+		"vendor": "com.snowplowanalytics.snowplow",
+		"name": "weather_enrichment_config",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+
+	"type": "object",
+	"properties": {
+		"vendor": {
+			"type": "string"
+		},
+		"name": {
+			"type": "string"
+		},
+		"enabled": {
+			"type": "boolean"
+		},
+		"parameters": {
+			"type": "object",
+			"properties": {
+				"apiKey": {
+                    "type": "string"
+				},
+                "cacheSize": {
+                    "type": "integer",
+                    "minimum": 1,
+                    "maximum": 1000000
+                },
+                "rounder": {
+                    "type": "integer",
+                    "minimum": 1,
+                    "maximum": 10
+                },
+                "host": {
```

Comment:
OWM uses several API hosts., as they explained it to me.
api.openweathermap.org - primary, free server. we can't do history lookups for it
pro.openweathermap.org - paid, SSL can be used, but little bit slower even in plain http;
history.openweathermap.org - history only, fast, doesn't support SSL.